kits/arm/ex_AR_kit.py

```diff
@@ -94,13 +105,13 @@
         rot_phone_init = R.from_quat(xyzw).as_matrix()
 
     # B6 - Grav Comp Mode
-    if m.get_button_diff(6) == 1:  # "ToOn"
+    if m.get_button_diff(3) == 1:  # "ToOn"
         m.set_led_color("blue")
         run_mode = "grav_comp"
         arm.cancel_goal()
 
     # B8 - Quit
-    if m.get_button_diff(8) == 1:  # "ToOn"
+    if m.get_button_diff(4) == 1:  # "ToOn"
```

Comment:
Not something we have to keep, but I think most examples have B8 as quit.  I think the consistency is nice here...any reason you changed this one to B4?